_tests_/reduck-tests/duckMethods.js

```diff
@@ -0,0 +1,147 @@
+/* Tests for the duck instance methods */
+import _ from 'lodash'
```

Comment:
So there's two `lodash` imports but `lodash` isn't declared as a dev dependency?

Also, is `lodash` really needed? `find`, `filter`, `map` exist on `Array.prototype` already and `assign` exists on `Object.prototype`.

Reply:
It is a dependency of a different package...I need to install it. I originally had vanilla javascript but I didn't like how long the files were getting, but I can use built in methods if you prefer.

Reply:
Hmm...i'm just thinking it might be a bit confusing to import a library we're not depending on directly. We should probably be explicit about whatever we're using to save any potential confusion. If you feel like those specific functions are needed to make the code more readable, lodash allows us to install any of its functions as a single npm package, for example `npm install lodash.find` and can be used directly like `import find from 'lodash.find'` with a lesser footprint on the overall package size.

Reply:
I didn't know that! I will see how many different methods I used and then decided if I should get rid of it all together or not.